### DISCUSSION TITLE
Windows-10-Sophia-Script with a GUI by default

### DISCUSSION THREAD

**Discussion by mryujitanaka** [farag2/Sophia-Script-for-Windows#113] [Q&A] [answered]
Any chance of ever having a version of Windows-10-Sophia-Script with a GUI by default instead of waiting for the benchtweakgaming.com team to always have to update their version of this project when you update your script?

I think you guys have the same goals, but you are moving in different directions. For the end user, it would be more beneficial if there was a more synchronized partnership.

What do you think about that?

**Comment by farag2** [comment, marked as answer]
Hello! The development is located here: https://github.com/farag2/SophiApp
May be the first public alpha in March. But no ETA. Screenshorts are totally outdated. Github Actions compiles a new release every Saturday. But for internal usage only. We have much work to do this year!

**Reply by kiquenet** [reply]
https://github.com/farag2/SophiApp nof found

**Reply by farag2** [reply]
Move here: https://github.com/SophiaUI
